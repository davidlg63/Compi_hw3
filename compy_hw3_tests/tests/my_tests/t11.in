bool foo( byte v,bool d){
    while(true){
        continue;
        break;
    }
}

void main(){
    foo = false;
}
