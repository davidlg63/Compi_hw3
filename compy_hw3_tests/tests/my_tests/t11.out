---end scope---
---end scope---
---end scope---
v BYTE -1
d BOOL -2
line 9: variable foo is not defined
